packages/common/src/groups/_internal/computeProps.ts

```diff
@@ -47,6 +47,8 @@ export function computeProps(
     "updateitemcontrol"
   );
 
+  hubGroup.memberType = group.userMembership.memberType;
```

Comment:
You might want to add some protection here, e.g. `getProp(group.userMembership, "memberType")` or `group.userMembership?.memberType`. The [API docs](https://developers.arcgis.com/rest/users-groups-and-items/group.htm) make it sound like `userMembership` is only returned on the response for authenticated users.